lib/transmuxer.js

```diff
@@ -1009,13 +1102,33 @@ Transmuxer = function() {
     self = this,
     videoTrack,
     audioTrack,
-    config,
-    pps,
+    earliestDTS,
 
     packetStream, parseStream, elementaryStream,
     aacStream, h264Stream,
     videoSegmentStream, audioSegmentStream;
 
+  this.dataEmitter = {
```

Comment:
Is this intended to be accessed publicly? Seems like it should be scoped to the closure or at least underline-suffixed.